miso-web/src/main/webapp/styles/style.css

```diff
@@ -1057,6 +1057,14 @@ table td.headertable {
     cursor: pointer;
 }
 
+td.headertable {
+    font-size: 50px;
+}
+
+td.headertable img {
+	vertical-align: top;
+}
+

```

Comment:
What do you think of the following styling? 
- Wrap the `${initparam['miso.name']}` in a div#instanceName (or whatever name)
- ```td.headertable #instanceName {
  font: Helvetica;
  padding: 0 30px;
  color: #666666;
  display: inline;
  font-size: 40px;
}```